src/improvements/tasks/MultisigTask.sol

```diff
@@ -409,8 +413,9 @@ abstract contract MultisigTask is Test, Script, StateOverrideManager {
         );
 
         string memory base64Json = Base64.encode(bytes(json));
+        printTitle("OP-TXVERIFY LINK");
         console.log(
-            "\nTo verify this transaction, run `op-txverify qr` on your machine, then open the following link on your mobile device: https://op-txverify.optimism.io/?tx=%s",
+            "To verify this transaction, run `op-txverify qr` on your machine, then open the following link on your mobile device: https://op-txverify.optimism.io/?tx=%s",
```

Comment:
How do signers install op-txverify, given that it's in a private repo?

Reply:
Need to figure that out still. I think it's okay to turn this feature on even if the worst case scenario happened where we didn't have signers use it for U15.